### DISCUSSION TITLE
[Feature] <Allow for shift+left click to select multiple photos>

### DISCUSSION THREAD

**Discussion by Tmatz27** [immich-app/immich#8049] [Feature Request]
### The feature

When i search for a photo or use the filter option i can only single click photos i want to grab. This can be tetious if im trying to grab for that a few pictures. It would be nice if there was an option to select all, select multiple with a shfit+left click, or something similar. 

I wanted to create a album based on a filter i had and it was to tedious to do so i didnt even attempt it. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1659
